src/classwork/04_assign/main.cpp

```diff
@@ -17,18 +18,25 @@ int main()
 	int num;
 	int total;
 	char choice = 'y';
-	while (choice == 'y' || choice == 'Y')
+	do
 	{
 		cout <<"Enter a number netween 1 and 10: ", '\n';
 		cin>>num;
+		if (num < 1 && num > 10)
+			{
+			cout<<"Not a valid number";
+			}
```

Comment:
Should be:
`while (num < 1 || num > 10)
			{
			cout<<"Not a valid number, try again Enter number from 1 to 10 ";
			cin>>num;
			}`